### PR TITLE
Fat 3571 c2339

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-3571
***
https://foliotest.testrail.io//index.php?/cases/view/2339
***

### CODE REVIEW

File: cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const selectActionProfile = Select({ name:'profile.action' });
+
+export default {
+  save:() => cy.do(Button('Save as profile & Close').click()),
+  verifyScreenName:(profileName) => cy.expect(Form(including(`Edit ${profileName}`)).exists()),
+
+  changeExistingInstanceRecordField:() => {
+    cy.do(Button({ id:'criterion-value-type' }).click());
+    cy.expect(SelectionList({ id: 'sl-container-criterion-value-type' }).exists());
+    cy.do(SelectionList({ id:'sl-container-criterion-value-type' })
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js:20]: move to const

**Review by IhorBohdan — APPROVED** [review]

File: cypress/support/fragments/data_import/match_profiles/matchProfileView.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Button,
+  Pane,
+  Accordion
+} from '../../../../../interactors';
+
+const viewPane = Pane({ id:'view-match-profile-pane' });
+
+export default {
+  edit:() => {
+    cy.wait(1500);
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/data_import/match_profiles/matchProfileView.js:12]: add comment for reason of this waiter pls

File: cypress/support/fragments/data_import/match_profiles/matchProfileView.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    cy.do(viewPane.find(Button('Actions')).click());
+    cy.do(Button('Edit').click());
+    cy.wait(1000);
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/data_import/match_profiles/matchProfileView.js:15]: add comment for reason of this waiter pls

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/match_profiles/matchProfileEdit.js]: Done

File: cypress/support/fragments/data_import/match_profiles/matchProfileView.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/match_profiles/matchProfileView.js:13]: Done

File: cypress/support/fragments/data_import/match_profiles/matchProfileView.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/match_profiles/matchProfileView.js:17]: Done

**Pull request merged**